utils/sql/db_update_manifest.txt

```diff
@@ -481,6 +481,7 @@
 9225|2023_01_21_bots_raid_members.sql|SHOW COLUMNS FROM `raid_members` LIKE 'bot_id'|empty|
 9226|2023_03_17_corpse_fields.sql|SHOW COLUMNS FROM `character_corpse_items` LIKE 'custom_data'|empty|
 9227|2023_03_24_npc_scale_global_base_verify.sql|SHOW COLUMNS FROM `npc_scale_global_base` LIKE 'heroic_strikethrough'|not_empty|
+9228|2023_04_15_update_resurrection_effects_block.sql|SELECT `rule_name` FROM `rule_values` WHERE `rule_name` LIKE 'Spells:ResurrectionEffectsBlock'|empty|
```

Comment:
This migration is unnecessary because we inject rules automatically. My suggestion is to slightly rename the rule so the old one will be automatically deleted and the desired default and notes will be automatically injected. Remove this migration, version.h entry and the .sql

Reply:
I originally had exactly that, rule renamed to "ResurrectionEffectBlock" but was informed to change it to a conversion to honor operators current setting.

Do we want to for sure rename to set the new default or leave the check so operators maintain the same setting currently and new builds default?

The query would have the same effect as the original PR just with catching edge cases that still allowed Resurrection Sickness to be overwritten. By changing and getting the new default it takes on the new functionality of this PR which is allowing both to be currently active and moving the buff to a new slot instead of being blocked completely by Resurrection Sickness.

I doubt anyone really has this original rule disabled to allow Resurrection Sickness overwrites but you never know.

Reply:
I agree.